packages/linter/src/rules/NoIconFontIsUsed.ts

```diff
@@ -35,7 +43,7 @@ export class NoIconFontIsUsed extends Rule {
   run({ $ }: Context) {
     // check for known classnames
     const iconFontCandidates = ICON_FONT_IDENTIFIERS.filter((identifier) => {
-      return $(`[class*=${identifier.className}]`).length > 0;
+      return $(`[class*=${identifier.className} i]`).length > 0;
```

Comment:
Out of curiosity, what does the `i` do here?

Reply:
case sensitivity?

Reply:
Exactly :)